src/lib/err/backtrace.c

```diff
@@ -115,7 +115,7 @@ clean_backtrace(void **stack, size_t depth, const ucontext_t *ctx)
  * that with a backtrace log.  Send messages via the tor_log function at
  * logger". */
 void
-log_backtrace_impl(int severity, int domain, const char *msg,
+log_backtrace_impl(int severity, uint64_t domain, const char *msg,
```

Comment:
Why aren't we using log_domain_mask_t here?
Does it break our may include rules?
If so, can you add a comment about keeping them in sync?

Reply:
Or can we put the type in a low-level header?